vertical-pod-autoscaler/recommender/Makefile

```diff
@@ -0,0 +1,36 @@
+all: build
+
+TAG?=0.0.1
+REGISTRY?=gcr.io/karol-kubernetes
+FLAGS=
+ENVVAR=
+GOOS?=linux
+
+deps:
+	go get github.com/tools/godep
+
+build: clean deps
+	$(ENVVAR) GOOS=$(GOOS) godep go build ./...
+	$(ENVVAR) GOOS=$(GOOS) godep go build -o recommender
+
+test-unit: clean deps build
+	$(ENVVAR) godep go test --test.short -race ./... $(FLAGS)
+
+docker:
+ifndef REGISTRY
+	ERR = $(error REGISTRY is undefined)
+	$(ERR)
+endif
+	docker build --pull -t ${REGISTRY}/recommender:${TAG} .
+	gcloud docker -- push ${REGISTRY}/recommender:${TAG}
+
+release: build docker
+
+clean:
+	rm -f recommender
```

Comment:
-rf

Reply:
it is a single binary. do we really need `-r` (reccurent) ?